### ISSUE TITLE
Improper heading structure on Events page

### ISSUE THREAD

**Issue by bbsmooth** [freeCodeCamp/chapter#2046]
### Describe the bug:
The Events page has `h3` headings nested within `h4` headings which is a WCAG failure of [SC 1.3.1](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html).

### Solution
I'm guessing that the `h4`s are supposed to be `div`s and this is just a mistake. 

Also, in a perfect world the `h3`s would be `h2`s so that heading levels aren't skipped.

**Comment by allella** [comment]
This is on /events

Example below
![image](https://user-images.githubusercontent.com/1777776/205754547-615547aa-c1af-4a53-9609-c0acc29505b7.png)

**Comment by Sboonny** [comment]
I think #1985 closes this, if I mistook, please open this and assign it to me 👍.
